test/_external/test262/wrapper.js

```diff
@@ -0,0 +1,2 @@
+require = require("esm")(module, { cjs: false, mode: "all" })
+module.exports = require("./test.js")
```

Comment:
Can't use `esm` or that's the version of `esm` included with `ava`.
You need to usr `../../../index.js` or so.